### DISCUSSION TITLE
Failed pods in quickstart setup

### DISCUSSION THREAD

**Discussion by mohsinansarirepo** [egovernments/Digit-Core#82] [DIGIT Platform]
Few months back quickstart setup was done, it was running fine with no issues but now quickstart setup on different machine is giving issues with 5 failed pods namely egov-user, egov-enc-service, egov-indexer, egov-persister and egov-persister. Below are some logs

I0514 07:34:36.340479 1 main.go:270] starting up: ["/git-sync"]
E0514 07:34:37.176641 1 main.go:294] error syncing repo: error running command: exit status 128: "Cloning into '/work-dir'...\nHost key verification failed.\r\nfatal: Could not read from remote repository.\n\nPlease make sure you have the correct access rights\nand the repository exists.\n"

**Comment by pradeepkumarcm-egov** [comment]
The same issue addressed in https://github.com/egovernments/DIGIT-DevOps/issues/1647
